spec/models/degree_spec.rb

```diff
@@ -36,8 +34,30 @@
         expect(subject).to validate_presence_of(:subject).on(:non_uk)
         expect(subject).to validate_presence_of(:country).on(:non_uk)
         expect(subject).to validate_presence_of(:graduation_year).on(:non_uk)
-        expect(subject).to validate_inclusion_of(:graduation_year)
-          .in_range(1900..Time.zone.today.year).on(:non_uk)
+      end
+    end
+
+    describe "custom validations" do
+      context "when graduation year is over 50 years in the past" do
```

Comment:
Since it's no longer 50 years and it could get adjusted in the future, lets change the context to "when graduation year is beyond the max limit" or something to that effect.

Reply:
fixed!